src/checkers/lemmeknow_checker.rs

```diff
@@ -4,7 +4,7 @@ use lemmeknow::{Data, Identify};
 use super::checker_type::{Check, Checker};
 
 const IDENTIFIER: Identify = Identify {
-    min_rarity: None,
+    min_rarity: Some(0.1),
```

Comment:
Why some here 🤔

Reply:
That is how `lemmeknow` works, `min_rarity` is an `Option<f32>` so that while filtering the JSON data, if we have it set to `None` we do nothing, else if it's `Some(val)`, we use `filter` on iterator to filter out values > val. So I think even if it's `0`, it _might_ perform some unnecessary checks.